lib/generator.js

```diff
@@ -835,6 +847,43 @@ class Generator {
       if (!isDirEmpty) throw new Error(`"${this.targetDir}" is not an empty directory. You might override your work. To skip this rule, please make your code a git repository or use the --force-write flag (not recommended).`);
     }
   }
+
+  /**
+   * Creates a custom resolver that maps mapBaseUrlToFolder.url to mapBaseUrlToFolder.folder
+   *
+   * @private
+   * @param  {string} fileName Name of the file to check against a list of glob patterns.
+   * @return {{read(*, *, *): Promise<unknown>, canRead(*): boolean, order: number}}
+   */
+  static getMapBaseUrlToFolderResolver(urlToFolder) {
```

Comment:
I don't think this method should be here. It's not particularly specific to the Generator class. It could actually be a separate package or a helper. Anyway, not a blocker for me, just my two cents.

Reply:
I don't remember previous code, but this function wasn't static 😅 I agree with Fran, but also for me this isn't a blocker.

Reply:
@fmvilas @magicmatatjahu you mean it should be moved to `utils.js` ? yup, makes sense

most important are updated to docs

Reply:
Yep, as I remember this function was integrate with instance of Generator, not definition of Generator and performed logic using `this.mapBaseUrlToFolder`. With current logic, we should move it to `utils.js`, but as I said, it's not a blocker for me, someone from community can handle it in followup as `good first issue` :)

Reply:
I did not want to have this as member function. Moving to util is a much better solution, thanks!